app/controllers/fines_controller.rb

```diff
@@ -31,4 +44,32 @@ def payments_response
   def payments
     (Hash.from_xml(Nokogiri::XML(payments_response).to_s) || {}).dig('LookupPatronInfoResponse', 'feeInfo')
   end
+
+  def fines_params
+    {
+      reason: fines.map(&:status).join(','),
+      billseq: fines.map(&:sequence).values_at(0, -1).join('-'),
+      amount: format('%.2f', fines.sum(&:owed))
+    }
+  end
+
+  def user_params
+    {
+      session_id: symphony_client.session_token,
```

Comment:
This is exposed to the user? We probably don't want to leak our session token to them or they'll be able to connect to symphony as our user, right?

Reply:
Now just using `SecureRandom.uuid`. The session id used to be the internal symphony Webcat session but can be anything and just has to exist unless we change the `payment_form` code.

Reply:
Now using just a string 'UNUSED'